src/components/footer/Footer.tsx

```diff
@@ -80,6 +81,7 @@ const Footer = ({
           {renderWelcomeScreen && <welcomeScreenHelpHintTunnel.Out />}
           <HelpButton
             onClick={() => actionManager.executeAction(actionShortcuts)}
+            title={t("helpDialog.title")}
```

Comment:
I think the default value of `title` prop should be `helpDialog.title`
cc/ @dwelle